boards/risc-v/esp32c3/esp32c3-devkit/kernel/esp32c3_userspace.c

```diff
@@ -59,15 +59,15 @@
  *    of _data.  like:  uint32_t *pdata = &_sdata;
  */
 
-extern uint32_t _stext;           /* Start of .text */
-extern uint32_t _etext;           /* End+1 of .text + .rodata */
-extern const uint32_t _eronly;    /* End+1 of read only section */
-extern uint32_t _sdata;           /* Start of .data */
-extern uint32_t _edata;           /* End+1 of .data */
-extern uint32_t _sbss;            /* Start of .bss */
-extern uint32_t _ebss;            /* End+1 of .bss */
+extern uint8_t _stext[];           /* Start of .text */
+extern uint8_t _etext[];           /* End_1 of .text + .rodata */
+extern const uint8_t _eronly[];    /* End+1 of read only section (.text + .rodata) */
+extern uint8_t _sdata[];           /* Start of .data */
+extern uint8_t _edata[];           /* End+1 of .data */
+extern uint8_t _sbss[];            /* Start of .bss */
+extern uint8_t _ebss[];            /* End+1 of .bss */
 
-extern uintptr_t *__ld_udram_end; /* End+1 of user ram section */
+extern uint8_t __ld_usram_end[];   /* End+1 of user ram section */
```

Comment:
```suggestion
extern uint8_t __ld_udram_end[];   /* End+1 of user ram section */
```
CI complained about this one.

Reply:
Done